web/gui/dashboard_info.js

```diff
@@ -2789,6 +2795,104 @@ netdataDashboard.context = {
         '<b>Metadata</b> is bcache\'s metadata overhead. '
     },
 
+    // ------------------------------------------------------------------------
+    // NFS client
+
+    'nfs.net': {
+        info: 'The number of received UDP and TCP packets.'
+    },
+
+    'nfs.rpc': {
+        info: '<p>Remote Procedure Call (RPC) statistics.</p>'+
+        '</p><b>Calls</b> - all RPC calls. '+
+        '<b>Retransmits</b> - retransmitted calls. '+
+        '<b>AuthRefresh</b> - authentication refresh calls (validating credentials with the server).</p>'
+    },
+
+    'nfs.proc2': {
+        info: 'NFSv2 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1094#section-2.2" target="_blank">RFC1094</a>.'
+    },
+
+    'nfs.proc3': {
+        info: 'NFSv3 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
+    },
+
+    'nfs.proc4': {
+        info: 'NFSv4 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc3010#section-14" target="_blank">RFC3010</a>.'
```

Comment:
rfc3010 is obsoleted by rfc3050 which is obsoleted by rfc7530 and does not include quite a lot of v4.1 operations from rfc8881.